projects/plugins/jetpack/modules/sharedaddy/sharing-sources.php

```diff
@@ -521,13 +521,13 @@ public function display_preview( $echo = true, $force_smart = false, $button_sty
 			esc_attr(
 				$is_deprecated
 					/* translators: %1$s is the name of a deprecated Sharing Service like "Google+" */
-					? sprintf( __( 'The %1$s service has shut down. This sharing button is not displayed to your visitors and should be removed.', 'jetpack' ), $this->get_name() )
+					? sprintf( __( 'The %1$s sharing service is no longer supported. This sharing button is not displayed to your visitors and should be removed.', 'jetpack' ), $this->get_name() )
```

Comment:
> I wonder whether "is no longer supported" is the best wording.

Yeah, I can see how this could be a bit confusing.

I'm a bit on the fence with adding a new method just to handle different wording though. Since we do not use that class that often, maybe we could come up with a more neutral / general wording that would work for everyone?

Reply:
Who can we ask for help with that wording? I don't have any ideas beyond something kind of vague like "The %1$s service has shut down or discontinued support for sharing buttons".

Reply:
I think "is no longer supported" would be fine - while I see the confusion that some users, may have, at the end of the day it's still just no longer supported regardless of which end is responsible.

But to expand on Brad's idea, maybe simply "The %1$s sharing service is no longer supported or has been discontinued by the provider." would do it?

Reply:
Either would work for me. For the title attribute we can afford to be long. It's probably trickier for the actual display, which has to be a lot shorter (currently `%1$s is no longer supported`):

<img width="556" alt="image" src="https://github.com/Automattic/jetpack/assets/426388/352cddc3-0084-4e45-a3fc-50ed9e82b53d">

What if we just said "is deprecated"?

Reply:
I don't think "is deprecated" is any better than "is no longer supported". Maybe worse if fewer blog admins even know what "deprecated" means, how common is the word outside of software development?

OTOH, if the tooltip says "The %1$s service has shut down or discontinued support for sharing buttons" then the button itself saying "is no longer supported" or the like seems less likely to cause lasting confusion.

Reply:
👍 I pushed c140cd2b53555fc4bcb3e8e47d2d1d247ff8fc47, so "is no longer supported" remains.